src/js/utils/browser.js

```diff
@@ -0,0 +1,69 @@
+import document from 'global/document';
+import window from 'global/window';
+
+const USER_AGENT = window.navigator.userAgent;
```

Comment:
there are a few other uses of the user agent, should we change those to use `browser.USER_AGENT` instead, maybe?